readme.md

```diff
@@ -44,6 +44,31 @@ javascript: (function() {
 
 You can use Git History for local git repos with the [CLI](https://github.com/pomber/git-history/tree/master/cli) or with the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=pomber.git-file-history).
 
+## Hacking
+
+To run from local checkout:
+
+    yarn install
+    yarn start
+
+it'll open the starting page in browser, you'll want to append the path that comes after `https://github.com/`
```

Comment:
I didn't complete formulating this sentence perhaps.
But the reason this needs clarification in the first place, is that starting page always says "2. Replace github.com with github.githistory.xyz".  Ideally it'd use current domain, e.g. "2. Replace github.com with localhost:3000" or "2. Replace github.com with deploy-preview-139--github-history.netlify.com".